tutorials/3d/portals/advanced_room_and_portal_usage.rst

```diff
@@ -104,3 +104,29 @@ The tent is a simple room inside a terrain room (which contains the ground, the
 This is perfect for improving performance in open world games. Often your buildings can be scenes (including the rooms and portals) that can be reused. When viewed from the outside, interiors will mostly be culled, and when viewing from the inside other buildings and most of the outside will be culled. The same goes for other players and objects that are inside and outside the buildings.
 
 *Scene is 'Diorama Eco scene' by Odo, with slight changes for illustration purposes.* `CC Attribution <https://creativecommons.org/licenses/by/4.0/>`_
+
+Internal room scenes
+^^^^^^^^^^^^^^^^^^^^
+Let us look in detail at another practical example for an open world. We want to place houses (as internal rooms) on an island, but have each house as a self contained scene containing both the interior *and* the external mesh of the house.
+
+.. image:: img/house_scene.png
+
+We have created a Room node (which will become the internal room) into which we have placed the interior meshes, and a Portal with no links (so autolinking will be used). The exterior mesh is *not* within the room. It will be autoplaced, and we are intending for it to be placed within the outer room.
```

Comment:
```suggestion
We have created a Room node (which will become the internal room) into which we have placed the interior meshes. We have also created a Portal with no links (so autolinking will be used). The exterior mesh is *not* within the room. It will be autoplaced, and we are intending for it to be placed within the outer room.
```